src/pages/Environments.js

```diff
@@ -129,17 +128,14 @@ const Environments = () => {
 
   const refreshData = Utils.withBusyState(setLoading, async () => {
     const creator = getUser().email
-    const [newRuntimes, newDisks, galaxyDisks, newApps] = await Promise.all([
+    const [newRuntimes, newDisks, newApps] = await Promise.all([
       Ajax(signal).Runtimes.list({ creator }),
       Ajax(signal).Disks.list({ creator }),
-      Ajax(signal).Disks.list({ creator, saturnApplication: 'galaxy' }),
       Ajax(signal).Apps.listWithoutProject({ creator })
     ])
```

Comment:
@kyuksel and @salonishah11 So it turns out that instead of relying on formattedBy, I could ask Disks.list to include the saturnApplication label in the return value-- I just found out about this by looking at https://github.com/DataBiosphere/terra-ui/pull/2662/files#diff-db07f65dd0ba5edbd535b2a231709c1f5901f4e1c43f39c333165d97c1abacc4R133

So I could still have a single Disks.list call like this:
`Ajax(signal).Disks.list({ creator, includeLabels: 'saturnApplication'}),`

And then apps would have their appType as label.saturnApplication (because we set that value at creation time), and I wouldn't need to use formattedBy to tell if multiple disks exist for the same app. And this approach does not have the current limitation of formattedBy not being set yet for disks that are in the Creating status.

Example disk info:
auditInfo: {creator: 'cahrens@gmail.com', createdDate: '2021-12-01T19:37:31.391603Z', destroyedDate: null, dateAccessed: '2021-12-01T19:37:32.255Z'}
blockSize: 4096
cloudContext: {cloudProvider: 'GCP', cloudResource: 'terra-test-f828b4cd'}
diskType: "pd-standard"
formattedBy: "CROMWELL"
googleProject: "terra-test-f828b4cd"
id: 14
labels: {saturnApplication: 'CROMWELL'}
name: "saturn-pd-141a3db4-6ce6-4896-b61b-6e85fdf38608"
size: 500
status: "Ready"
zone: "us-central1-a"

Reply:
The ability to label disks was added recently (by [this PR](https://github.com/DataBiosphere/leonardo/pull/2340)) so we may not be able to rely on it for disks created prior to then, for backwards compatibility?

Reply:
I did not know that labels were a recent thing. `formattedBy` seems like the better solution because it allows us to know who formatted non-app disks, but thus far the assumption has just been that there is only a single non-app disk that we care about (since Jupyter and RStudio can't exist in the same workspace at the same time). I'll wait until tomorrow and check in with others before making a decision about whether to stick with formattedBy or go with labels.

Reply:
Is `formattedBy` populated for runtime (`Jupyter` | `RStudio`) disks? Even if it isn't, it should be okay to make the assumption that disk belongs to a runtime when `formattedBy` is `undefined`.

Reply:
`formattedBy` is "GCE" for Jupyter (and I assume RStudio). I have seen it be null in two cases-- when the disk is in "Creating" (always), and once when I tried creating a Galaxy instance and it errored due to permissions.

In the latter case, I got a Galaxy instance with an error indication on the Cloud Environments page, and the disk with null formattedBy. After I deleted the Galaxy instance, I was unable to delete the disk… if I remember correctly, delete was disabled because it said that I needed to delete the cloud environment it was attached to (even though there was none left). However, my fiab is no longer in that state and I don't recall what I later did.

I am trying to reproduce the bug where Jupyter tries to attached to Galaxy's disk in dev. I am pretty sure I saw it back when I was giving the Cromwell demo to the IA team, but it is tricky to reproduce because if I understand the cause of the problem, Jupyter has to try to get a disk instance while Galaxy's is still in "Creating".

Reply:
I have gone back to using the saturnApplication label (though still with just a single call to Disks.list).

Reply:
[Slack thread](https://broadinstitute.slack.com/archives/C02H8KALYMD/p1638450691023700) for posterity